components/script/lib.rs

```diff
@@ -9,17 +9,12 @@
 #![feature(register_tool)]
 #![deny(unsafe_code)]
 #![doc = "The script crate contains all matters DOM."]
-#![cfg_attr(
-    not(any(
-        feature = "unrooted_must_root_lint",
-        feature = "trace_in_no_trace_lint"
-    )),
-    allow(unknown_lints)
-)]
-#![allow(deprecated)] // FIXME: Can we make `allow` only apply to the `plugin` crate attribute?
-#![plugin(script_plugins)]
-#![register_tool(unrooted_must_root_lint)]
-#![register_tool(trace_in_no_trace_lint)]
+#![register_tool(crown)]
+
+// issue a warning if there is no crown
+#[warn(unknown_lints)]
+#[deny(crown_is_not_used)]
+// comment to keep tidy happy
```

Comment:
What's the issue here? It might be worth opening a bug for this and linking to it.

Reply:
I meant in relation to the "comment to keep tidy happy" comment. It's unclear what tidy is complaining about here.

Reply:
Something about redundant new line, but I would like to keep the new line to keep warn/deny separated from next statement (as it has nothing to do with next statement).

Reply:
Fixed